### ISSUE TITLE
Fungal infection of the nail

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10040]
**Preferred term label:**
Fungal infection of the nail

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Favus

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this similar e3nough to 
[Chronic monilial nail infection](https://hpo.jax.org/app/browse/term/HP:0008396)
(which I believe is limited to candida)?

**Comment by MickeySegal** [comment]
OK.

**Comment by pnrobinson** [comment]
OK, closing, will add as related synonym
